README.md

```diff
@@ -69,6 +69,7 @@ A dbt profile can be configured to run against Trino using the following configu
 | threads                        | How many threads dbt should use                                                                              | Optional (default is `1`)                                                                               | `8`                              |
 | prepared_statements_enabled    | Enable usage of Trino prepared statements (used in `dbt seed` commands)                                      | Optional (default is `true`)                                                                            | `true` or `false`                |
 | retries                        | Configure how many times a database operation is retried when connection issues arise                        | Optional (default is `3`)                                                                               | `10`                             |
+| timezone                       | The time zone for the session                                                                                | Optional (defaults to the client side local timezone)                                                   | `Europe/Brussels`                |
```

Comment:
optional: `The time zone for the session` -> `The time zone for the Trino session`

Reply:
changed